cli/src/pcluster/resources/head_node/user_data.sh

```diff
@@ -94,10 +94,10 @@ function vendor_cookbook
 # deploy config files
 export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/aws/bin
 # Load ParallelCluster environment variables
-[ -f /etc/profile.d/pcluster.sh ] && . /etc/profile.d/pcluster.sh
+[ -f /etc/profile.d/cookbook_environment.sh ] && . /etc/profile.d/cookbook_environment.sh && echo $CFN_BOOTSTRAP_VIRTUALENV_PATH
```

Comment:
I think echoing the envar here was for debugging purpose. Can we remove the echo?